.gitignore

```diff
@@ -85,7 +85,7 @@ docs/*/pdfreactor-fonts.html
 #################
 *.egg-info
 .installed.cfg
-static/
+/static/
```

Comment:
Pretty sure it's a relic of some Bad Old Days where developers needed to run collectstatic. As written, this rule was ignoreing *every* directory called 'static', which was getting in my way. There may not be a reason for this line at all, since there's not a top-level 'static' directory anyways.